tools/clean-emoji-cache

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+from __future__ import absolute_import
+from __future__ import print_function
+import os
+import subprocess
+import sys
+import hashlib
```

Comment:
yeah, I forgot to remove it after moving the `get_success_stamp` function to lib and start using it.